tests/func/main.rs

```diff
@@ -113,6 +132,7 @@ impl ServerGuard {
                                 .expect("valid server data");
                             tx.send((port, runstate_dir, tls_cert_file))
                                 .expect("valid channel");
+                            break;
```

Comment:
Sure. This still doesn't make the fix valid.

Reply:
> The proper way to avoid all this hackery here would be to switch to --emit-server-status.

> Sure. This still doesn't make the fix valid.

Yeah, I see.

@tailhook Paul, please work on switching to `--emit-server-status`. I, unfortunately, don't have time to look into this myself, pretty busy with lots of other stuff.